en_US/security/authn/authn.md

```diff
@@ -1,243 +1,818 @@
+    "salt_position":"prefix"
+  },"
+  user_id_type":"clientid"
+}
+```
 
+#### GET /api/v5/listeners/{listener_id}/authentication/{id}/status
 
+Get statistics of an authenticator from an MQTT listener authentication chain.
 
-## Authentication chain
+```bash
+## Request
+curl -i \
+--basic \
+-u admin:public \
+-X GET \
+http://localhost:18083/api/v5/listeners/tcp%3Adefault/authentication/password_based%3Abuilt_in_database/status
+
+## Response
+{
+  "metrics": {
+    "failed": 0,
+    "nomatch": 0,
+    "rate": 0,
+    "rate_last5m": 0,
+    "rate_max": 0,
+    "success": 0,
+    "total": 0
+  },
+  "node_error": [],
+  "node_metrics": [
+    {
+      "metrics": {
+        "failed": 0,
+        "nomatch": 0,
+        "rate": 0,
+        "rate_last5m": 0,
+        "rate_max": 0,
+        "success": 0,
+        "total": 0
+      },
+      "node": "emqx@127.0.0.1"
+    }
+  ],
+  "node_resource_metrics": [
+    {
+      "metrics": {},
+      "node": "emqx@127.0.0.1"
+    }
+  ],
+  "node_status": [
+    {
+      "node": "emqx@127.0.0.1",
+      "status": "connected"
+    }
+  ],
+  "resource_metrics": {},
+  "status": "connected"
+}
+```
 
-When multiple authentication methods are enabled at the same time, EMQX will perform chain authentication in the order in which the plugins are opened:
-- Once authentication succeeds, terminate the authentication chain and allow clients to access
-- Once authentication fails, terminate the authentication chain and prohibit client access
-- If Failing to pass until the last authentication method,  it is determined according to  **anonymous authentication** configuration
-  - Allow client access when anonymous authentication is enabled
-  - Deny client access when anonymous authentication is disabled
+#### POST /api/v5/listeners/{listener_id}/authentication/{id}/move
 
+Move an authenticator within an MQTT listener authentication chain.
 
+The following move commands are available:
 
-![_images/guide_2.png](./assets/guide_2.png)
+* Move an authenticator to the beginning of the chain (it will be applied first during authentication):
+  ```json
+  {"position": "front"}
+  ```
+* Move to the end of the chain (it will be applied last during authentication):
+  ```json
+  {"position": "rear"}
+  ```
+* Place at the position preceding some other authenticator:
+  ```json
+  {"position": "before:password_based:built_in_database"}
+  ```
 
-<!-- replace -->
+```bash
+## Request
+curl -i \
+--basic \
+-u admin:public \
+-X POST \
+-H "Content-Type: application/json" \
+http://localhost:18083/api/v5/listeners/tcp%3Adefault/authentication/password_based%3Abuilt_in_database/move \
+-d @- <<'EOF'
+{"position": "front"}
+EOF
+
+## Response
+## 204 No Content
+```
 
-::: tip
+## TLS authentication
 
-It can improve client authentication efficiency when enabling only one authentication plugin at the same time
+To enable TLS authentication for clients, one may add a `ssl` listener
+with `verify_peer` verify option set.
```

Comment:
It's very hard to understand the meaning of this phrase. Did you mean

```
To enable TLS authentication for clients, one may set `ssl.verify` option to `verify_peer` for the listener.
```

Reply:
I tried to simplify it.